cmd/proxy/actions/list_test.go

```diff
@@ -0,0 +1,11 @@
+package actions
+
+func (a *ActionSuite) TestList() {
+	r := a.Require()
+
+	req := a.JSON("/github.com/gomods/athens/@v/list")
```

Comment:
Let's have all repository examples be stemmed from `github.com/athens-artifacts`, this way we can actually test the response and be sure it won't change overtime :)